packages/runtime/container-runtime/src/id-compressor/idCompressor.ts

```diff
@@ -80,15 +79,21 @@ export class IdCompressor implements IIdCompressor, IIdCompressorCore {
 	// The gen count to be annotated on the range returned by the next call to `takeNextCreationRange`.
 	// This is updated to be equal to `generatedIdCount` + 1 each time it is called.
 	private nextRangeBaseGenCount = 1;
-	// The capacity of the next cluster to be created
-	private newClusterCapacity = initialClusterCapacity;
 	private readonly sessions = new Sessions();
 	private readonly finalSpace = new FinalSpace();
 
 	// -----------------------
 
-	// ----- Telemetry state -----
+	// ----- Ephemeral state -----
 
+	/**
+	 * Roughly equates to a minimum of 1M sessions before we start allocating 64 bit IDs.
```

Comment:
Nit: I still think the contents here belong in a `@remarks` block, and that the addition of a semantic description would be useful.